src/components/editor/plugins/floating-toolbar-kit.tsx

```
@@ -0,0 +1,18 @@
+export const FloatingToolbarKit = [
+  createPlatePlugin({
+    key: 'floating-toolbar',
+    render: {
+      afterEditable: () => (
+        <FloatingToolbar>
+          <FloatingToolbarButtons />
+        </FloatingToolbar>
+      ),
+    },
+  }),
+];
```

ESLint

[error] Insert `⏎`

(prettier/prettier)